tests/snuba/tagstore/test_tagstore_backend.py

```diff
@@ -1088,6 +1089,110 @@ def test_get_perf_group_list_tag_value_by_environment(self):
         }
 
 
+class ProfilingTagStorageTest(TestCase, SnubaTestCase, SearchIssueTestMixin):
+    def setUp(self):
+        super().setUp()
+        self.ts = SnubaTagStorage()
+
+    def test_get_profiling_groups_user_counts_simple(self):
+        first_group_fingerprint = f"{GroupType.PROFILE_BLOCKED_THREAD.value}-group1"
+        first_group_timestamp_start = timezone.now() - timedelta(days=5)
+
+        self.store_search_issue(
+            self.project.id,
+            1,
+            [first_group_fingerprint],
+            self.environment.name,
+            first_group_timestamp_start + timedelta(minutes=1),
+        )
+
+        self.store_search_issue(
+            self.project.id,
+            1,
+            [first_group_fingerprint],
+            self.environment.name,
+            first_group_timestamp_start + timedelta(minutes=2),
+        )
+
+        self.store_search_issue(
+            self.project.id,
+            2,
+            [first_group_fingerprint],
+            self.environment.name,
+            first_group_timestamp_start + timedelta(minutes=3),
+        )
+
+        event, issue_occurrence, group_info = self.store_search_issue(
+            self.project.id,
+            3,
+            [first_group_fingerprint],
+            None,
+            first_group_timestamp_start + timedelta(minutes=4),
+        )
+        first_group = group_info.group if group_info else None
+
+        second_group_fingerprint = f"{GroupType.PROFILE_BLOCKED_THREAD.value}-group2"
+        second_group_timestamp_start = timezone.now() - timedelta(hours=5)
+        for incr in range(1, 4):
+            event, issue_occurrence, group_info = self.store_search_issue(
+                self.project.id,
+                incr,
+                [second_group_fingerprint],
+                self.environment.name if incr != 4 else None,
```

Comment:
`incr` will never be 4 here, it'll be `1, 2, 3`

Reply:
Heh, nice catch. I think this was a copy paste bug.